doc/docs/usage/tutorial-vehicle-signals.md

```diff
@@ -449,6 +449,20 @@ After running one of these commands, the `speed-provider` workload has been upda
 
 You can verify this by re-opening the web UI on <http://127.0.0.1:5000>.
 
+### List configuration items
+
+Let's list the configuration items present in current state with the `ank get configs` command.
+
+Using `ank -k get configs`, it should print:
+
+```text
+CONFIGS
+network
+env_provider
+env_consumer
+agents
+```
+
 ### Delete configuration items
 
 Let's try to delete a configuration item still referenced by workloads in its `configs` field by re-using the previous manifest content.
```

Comment:
Can you please also update the the "Delete configuration items" section to use the new `ank delete config env_provider` instead of the `ank apply -d`.
Don't forget to delete the last sentence in this chapter, i.e.,  "This can be done in a single step." as doing this in a single step is not trivial and requires usage of `ank set state` with a proper config and update masks.

Reply:
I've updated the command used and removed the last sentence, also checked that the rest of documentation remains valid and it throws the same described error.